index.html

```diff
@@ -0,0 +1,175 @@
+        </div>
+    </main>
+    <!-- The Project Section -->
+    <section>
+        <h2>My Recent <br> Works</h2>
+        <div class="underline"></div>
+        <div class="section_container">
+        <article class="card">
+            <div class="img"></div>
+            <div class="details">
+            <h5>Multi-Post Stories Gain+Glory</h5>
+            <ul>
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>html</li>
+            </ul>
+            <button>See Project</button>
+        </div>
+        </article>
+        <article class="card">
+            <div class="img"></div>
+            <div class="details">
+            <h5>Multi-Post Stories Gain+Glory</h5>
+            <ul>
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>html</li>
+            </ul>
+            <button>See Project</button>
+        </div>
+        </article>
+        <article class="card">
+            <div class="img"></div>
+            <div class="details">
+            <h5>Multi-Post Stories Gain+Glory</h5>
+            <ul>
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>html</li>
+            </ul>
+            <button>See Project</button>
+        </div>
+        </article>
+        <article class="card">
+            <div class="img"></div>
+            <div class="details">
+            <h5>Multi-Post Stories Gain+Glory</h5>
+            <ul>
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>html</li>
+            </ul>
+            <button>See Project</button>
+        </div>
+        </article>
+        <article class="card">
+            <div class="img"></div>
+            <div class="details">
+            <h5>Multi-Post Stories Gain+Glory</h5>
+            <ul>
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>html</li>
+            </ul>
+            <button>See Project</button>
+        </div>
+        </article>
+        <article class="card">
+            <div class="img"></div>
+            <div class="details">
+            <h5>Multi-Post Stories Gain+Glory</h5>
+            <ul>
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>html</li>
+            </ul>
+            <button>See Project</button>
+        </div>
+        </article>
+        </div>
+    </section>
+    <!-- The About Article -->
+    <article class="about_me">
+    <div class="about_me_background">
+            <h2>About me</h2>
+      <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+        see and have a project you need coded,
```

Comment:
- [ ] Great job using semantic HTML code 👏 . Please wrap your `article` tag in a `section` tag as the about me is a new section of your webpage 🙏 .